app/views/courses/EnrollmentsView.coffee

```diff
@@ -102,6 +93,34 @@ module.exports = class EnrollmentsView extends RootView
     @state.set('totalCourses', @courses.size())
     super()
 
+  onPrepaidsSync: ->
+    @prepaids.each (prepaid) =>
+      prepaid.creator = new User()
+      # We never need this information if the user would be `me`
+      if prepaid.get('creator') isnt me.id
+        @supermodel.trackRequest prepaid.creator.fetchCreatorOfPrepaid(prepaid)
+
+    @decideUpsell()
+
+  onLeadPriorityResponse: ({ priority }) ->
+    @state.set({ leadPriority: priority })
+    @decideUpsell()
+
+  decideUpsell: ->
+    # There are also non classroom prepaids.  We only use the course or starter_license prepaids to determine
+    # if we should skip upsell (we ignore the others).
+
+    coursePrepaids = @prepaids.filter((p) => p.get('type') == 'course')
+
+    skipUpsellDueToExistingLicenses = coursePrepaids.length > 0
+    shouldUpsell = !skipUpsellDueToExistingLicenses and (@state.get('leadPriority') is 'low') and (me.get('preferredLanguage') isnt 'nl-BE')
```

Comment:
We should treat both Belgian- and Netherlands-flavored Dutch the same (`"nl-NL"` as well as `"nl=BE"`):
```suggestion
    shouldUpsell = !skipUpsellDueToExistingLicenses and (@state.get('leadPriority') is 'low') and not /^nl/.test(me.get('preferredLanguage', true))
```

Reply:
@jmif This feedback might have been missed.

Reply:
Thanks for pinging.  Didn't miss however I should have dropped a note here.  Was merging this quickly to get this merged in and planned to follow up this evening once I got around to doing so.

I've split this out into its own PR so that we could ship this change and then implement / test this suggestion separately.  This is open here https://github.com/codecombat/codecombat/pull/5289